tests/CbtThemeLocale/escapeBlockAttributes.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+require_once __DIR__ . '/base.php';
+
+/**
+ * Tests for the CBT_Theme_Locale::escape_text_content_of_blocks method with block attributes.
+ *
+ * @package Create_Block_Theme
+ * @covers CBT_Theme_Locale::escape_text_content_of_blocks
```

Comment:
The test covers both of these functions, right?

```suggestion
 * @covers CBT_Theme_Locale::escape_text_content_of_blocks
 * @covers CBT_Theme_Locale::escape_block_attribute_strings
```